### PR TITLE
chore: drop dependency on deprecated `gulp-util`

### LINKED ISSUE #235: Replace deprecated dependency gulp-util
[`gulp-util`](https://www.npmjs.com/package/gulp-util) has been recently deprecated. Continuing to use this dependency may prevent the use of your library with the latest release of Gulp 4 so **it is important to replace `gulp-util`**.

The [README.md](https://github.com/gulpjs/gulp-util) lists alternatives for all the components so a simple replacement should be enough.

Your package is one of the most popular packages still relying on `gulp-util`, it would be good to publish a fixed version to npm as soon as possible.

See:
- https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
- https://github.com/gulpjs/gulp-util/issues/143

### PR DESCRIPTION
**Pull request by TheDancingCode**
Closes #235

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,16 +27,19 @@
...
     "escape-string-regexp": "^1.0.5",
     "event-stream": "^3.1.0",
+    "fancy-log": "^1.3.2",
     "group-array": "^0.3.0",
-    "gulp-util": "^3.0.0",
+    "plugin-error": "^0.1.2",
     "stream-to-array": "^2.3.0",
-    "through2": "^2.0.1"
+    "through2": "^2.0.1",
+    "vinyl": "^2.1.0"
```
Comment (demurgos) [inline package.json]: `vinyl` should be in the `devDependencies`. Could you update it please?

**Comment by demurgos** [comment]
@klei Do you have an idea why CI is failing?

Could you look into this and merge/publish this PR?

**Comment by demurgos** [comment]
@TheDancingCode 
Thanks for the update.

**Comment by FizzyGalacticus** [comment]
Any update on this?

**Comment by TheDancingCode** [comment]
It's failing because of the NPM token that is required in `.npmrc`.

**Comment by demurgos** [comment]
@joakimbeng 
Could you look into this PR?

**Comment by rejas** [comment]
has @joakimbeng given up on this project? can somebody recommend an alternative to replace this lib in my projects?

**Comment by joakimbeng** [comment]
@rejas I haven't completely given up on the project, but I'm currently on parental leave and do almost no coding at all. Also I don't use Gulp anymore myself and therefore have no use of this plugin, which makes it less likely for me to update it frequently. I had written a statement about wanting help in the Readme, which I clumsily had removed in a merge but is now added again.

This plugin seems to have about 15k downloads per day so hopefully there's someone willing to become a core contributor/maintainer. If not I don't see myself giving this project as much attention as people want...

**Comment by rejas** [comment]
@joakimbeng I totally understand you and hope the paternal leave is working out great and you get some sleep nonetheless :-) 
Anyway, I could take over maintainership here (and in npm) if you want, still using gulp and this plugin.

**Pull request merged**